CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
+## 1.5.0
+
+* Add 'force' command to record the unapplied migrations from the filesystem in
```

Comment:
To me, "force" sounds like it should mean "apply this migration whether or not there's a record for it." Django's migration tool uses --fake for this purpose -- maybe we should use a similar word?